recipes/boost/all/conanfile.py

```diff
@@ -543,36 +539,6 @@ def add_defines(option, library):
         toolset, _, _ = self._get_toolset_version_and_exe()
         flags.append("toolset=%s" % toolset)
 
-        if self.settings.get_safe("compiler.cppstd"):
-            flags.append("cxxflags=%s" % cppstd_flag(
-                    self.settings.get_safe("compiler"),
-                    self.settings.get_safe("compiler.version"),
-                    self.settings.get_safe("compiler.cppstd")
-                )
-            )
```

Comment:
This code did not work as expected, at least when using apple-clang. I'll post the conan profile used in the main comments.